2_Operations/scrum.md

```diff
@@ -52,3 +52,13 @@ Sometimes the result of a user story is an agreement that we should do a certain
 There are a bunch of tasks we have to do every month/quarter/year. To make sure they get assigned enough time for proper execution, we include them in sprints as user stories. The only difference is, they don't need user story demo recordings since they happen so often and are repetitive.
 
 For every month/quarter that a recurring task is required we (re)open the issue and create a comment for the new month/quarter.
+
+## On-Calls
+
+Sometimes, things break in the middle of the night or on weekends. The great thing about being a [remote-first team](https://www.amazon.com/Remote-Office-Not-Required/dp/0804137501) is that we cover different time zones so there is almost always someone available. The solution for the weekend is to have Niteans on-call.
```

Comment:
I would avoid adding external links on pages like this because it's really hard to monitor.